plugins/woocommerce/tests/php/includes/class-wc-emails-tests.php

```diff
@@ -29,9 +30,33 @@ public function test_email_footer_is_compatible_with_legacy_do_action() {
 		// 10 is expected priority of the hook.
 		$this->assertEquals( 10, has_action( 'woocommerce_email_footer', array( $email_object, 'email_footer' ) ) );
 		ob_start();
+		// phpcs:ignore WooCommerce.Commenting.CommentHooks.MissingHookComment
 		do_action( 'woocommerce_email_footer' );
 		$content = ob_get_contents();
 		ob_end_clean();
 		$this->assertFalse( empty( $content ) );
 	}
+
+	/**
+	 * Test that order meta function outputs linked meta.
+	 */
+	public function test_order_meta() {
+		add_filter(
+			'woocommerce_email_order_meta_keys',
+			function () {
+				return array( 'dummy_key' );
+			}
+		);
+		$order = \Automattic\WooCommerce\RestApi\UnitTests\Helpers\OrderHelper::create_order();
+		$order->add_meta_data( 'dummy_key', 'dummy_meta_value' );
+		$order->save();
+
+		$email_object = new WC_Emails();
+		ob_start();
+		$email_object->order_meta( $order, true, true );
+		$content = ob_get_contents();
+		ob_end_clean();
```

Comment:
Minor/not essential, but we could simplify just a little:

```suggestion
		$content = ob_get_clean();
```